### PR TITLE
Nullable cleanup

### PR DESCRIPTION
**Pull request by JamesNK**
Addresses https://github.com/dotnet/aspnetcore/issues/5680

### CODE REVIEW

File: src/Tools/GetDocumentInsider/src/GetDocument.Insider.csproj [hunk 1]
```diff
@@ -7,7 +7,6 @@
     <RootNamespace>Microsoft.Extensions.ApiDescription.Tool</RootNamespace>
     <TargetFrameworks>netcoreapp2.1;$(DefaultNetFxTargetFramework)</TargetFrameworks>
     <IsShippingPackage>false</IsShippingPackage>
-    <Nullable>disable</Nullable>
```
Comment (JamesNK) [inline src/Tools/GetDocumentInsider/src/GetDocument.Insider.csproj:10]: props file in tools directory already disables nullability.

File: src/HttpClientFactory/Polly/src/Microsoft.Extensions.Http.Polly.csproj [hunk 2]
```diff
@@ -13,6 +13,7 @@
     <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
     <IsPackable>true</IsPackable>
     <IsShipping>true</IsShipping>
+    <!-- Nullable disabled because package targets netstandard2.0. -->
```
Comment (BrennanConroy) [inline src/HttpClientFactory/Polly/src/Microsoft.Extensions.Http.Polly.csproj:16]: We can enable nullable for netstandard projects
https://github.com/dotnet/aspnetcore/blob/b883b451b02876b9adaf3c4ce7ae0ac441f471bb/eng/targets/CSharp.Common.targets#L115

Comment (JamesNK) [reply src/HttpClientFactory/Polly/src/Microsoft.Extensions.Http.Polly.csproj]: Nice. TIL.

File: src/SiteExtensions/Microsoft.Web.Xdt.Extensions/src/Microsoft.Web.Xdt.Extensions.csproj [hunk 3]
```diff
@@ -6,6 +6,7 @@
     <CopySymbolsToArtifacts>true</CopySymbolsToArtifacts>
     <IsPackable>false</IsPackable>
     <IsShippingPackage>false</IsShippingPackage>
+    <!-- Nullable disabled because package only targets .NET Framework. -->
```
Comment (BrennanConroy) [inline src/SiteExtensions/Microsoft.Web.Xdt.Extensions/src/Microsoft.Web.Xdt.Extensions.csproj:9]: This one should be easy, it's a single file 😄

File: src/HttpClientFactory/Polly/src/Microsoft.Extensions.Http.Polly.csproj [hunk 4]
```diff
@@ -13,7 +13,6 @@
     <ExcludeFromSourceBuild>true</ExcludeFromSourceBuild>
     <IsPackable>true</IsPackable>
     <IsShipping>true</IsShipping>
-    <Nullable>disable</Nullable>
```
Comment (BrennanConroy) [inline src/HttpClientFactory/Polly/src/Microsoft.Extensions.Http.Polly.csproj:16]: You didn't actually annotate the polly library 😕 
For example, `HttpRequestMessageExtensions.GetPolicyExecutionContext` can return a null `Context` and `HttpRequestMessageExtensions.SetPolicyExecutionContext` can accept a null `Context`.

If we aren't going to annotate it yet, then we shouldn't remove this.

Comment (JamesNK) [reply src/HttpClientFactory/Polly/src/Microsoft.Extensions.Http.Polly.csproj:16]: Hmm, I didn't get any warnings after enabling nullable. I guess you can enable nullable for netstandard2.0, but there is no analysis.

Comment (JamesNK) [reply src/HttpClientFactory/Polly/src/Microsoft.Extensions.Http.Polly.csproj:16]: Done

**Review by BrennanConroy — APPROVED** [review]

**Review by captainsafia — APPROVED** [review]

**Pull request merged**